alerting/src/main/kotlin/org/opensearch/alerting/DocumentLevelMonitorRunner.kt

```diff
@@ -93,7 +97,9 @@ object DocumentReturningMonitorRunner : MonitorRunner() {
 
         try {
             val getAliasesRequest = GetAliasesRequest(index)
-            val getAliasesResponse = monitorCtx.client!!.admin().indices().getAliases(getAliasesRequest).actionGet()
+            val getAliasesResponse: GetAliasesResponse = monitorCtx.client!!.suspendUntil {
```

Comment:
It shouldnt, however since the monitor Runner class was refactored, `monitorCtx` get the client and its other attributes when those are registered to the `MonitorRunnerExecutionContext`. This is done [here](https://github.com/opensearch-project/alerting/blob/main/alerting/src/main/kotlin/org/opensearch/alerting/MonitorRunnerService.kt)

Reply:
You could make it a `lateinit var`. Otherwise, we should use `client?` and handle the null case since we should avoid using `!!` outside of tests.